bindings/py/setup.py

```diff
@@ -209,6 +210,10 @@ def generateExtensions():
   setup(
     name="nupic.bindings",
     version=getVersion(),
+    # This distribution contains platform-specific C++ libraries, but they are not
+    # built with distutils. So we must create a dummy Extension object so when we
+    # create a binary file it knows to make it platform-specific.
+    ext_modules=[Extension('nupic.dummy', sources = ['dummy.c'])], 
```

Comment:
@rhyolight Yep, that works great locally. Producing the expected platform specific bdist and bdist_wheel naming.